pyedr/pyedr/tests/datafiles.py

```diff
@@ -17,77 +17,78 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor,
 # Boston, MA  02110-1301  USA
 
-# Move to impportlib.resources when py>=3.7
-from pkg_resources import resource_filename
+from importlib import resources
 
 from pathlib import Path
 
 
-EDR = resource_filename(__name__, 'data/cat_small.edr')
-EDR_XVG = resource_filename(__name__, 'data/cat_small.xvg')
-EDR_UNITS = resource_filename(__name__, 'data/cat_small_units.p')
+_data_ref = resources.files('pyedr.tests.data')
 
-EDR_IRREG = resource_filename(__name__, 'data/irregular.edr')
-EDR_IRREG_XVG = resource_filename(__name__, 'data/irregular.xvg')
-EDR_IRREG_UNITS = resource_filename(__name__, 'data/irregular_units.p')
+EDR = (_data_ref / 'cat_small.edr').as_posix()
```

Comment:
Purely out of curiousity, would
```suggestion
EDR = str(_data_ref / 'cat_small.edr')
```
not wort for Windows-y reasons?

Reply:
If I remember correctly as_posix just returns str + a slahes replacement. I think I'm mostly being overly cautious - I suspect it would work fine either way.